README.md

```diff
@@ -25,6 +25,18 @@ Most things that you can do manually in the browser can be done using Puppeteer!
 
 Give it a spin: https://try-puppeteer.appspot.com/
 
+## Table of Contents
```

Comment:
let's use our TOC-generator - just add a <!-- GEN:TOC><!-- GEN:stop --> comments here instead of the actual TOC, and then run `npm run doc`

Reply:
The respective error seems to pop up ( with `npm run doc` ) even after the successful execution of `npm i` :thinking: 
```md
> puppeteer@1.18.1-post doc /home/james-the-hacker/CodeSpace/GitHub-Projects/puppeteer
> node utils/doclint/cli.js

(node:6073) UnhandledPromiseRejectionWarning: Error: Chromium revision is not downloaded. Run "npm install" or "yarn install"
    at Launcher.launch (/home/james-the-hacker/CodeSpace/GitHub-Projects/puppeteer/lib/Launcher.js:119:15)
    at <anonymous>
(node:6073) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:6073) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Reply:
@aslushnikov ??

Reply:
@jamesgeorge007 did you try running "npm i" like the error recommends?

Reply:
Seems to be an issue with my local environment. The issue still persists :confused:

Reply:
Fix